.github/workflows/validate-devfiles-minikube.yaml

```diff
@@ -37,3 +37,15 @@ jobs:
         odo: "latest"
     - name: Validate the devfile stacks
       run: tests/test.sh odo
+  check_architectures:
+    name: Check Stack and Sample Architectures
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Download yq
+      run: wget https://github.com/mikefarah/yq/releases/download/v4.9.8/yq_linux_amd64 -O ./yq && chmod +x ./yq
+    - name: Check stacks and samples
+      continue-on-error: true
```

Comment:
With this field set, is there a scenario in which `check_architectures.sh` could exit with an non-zero code, and we would want this check to still fail?

Reply:
so curently the script exits with non zero, if we dont put `continue-on-error: true`, all the builds will finish as ❌  which really doesn't look great..

![Screen Shot 2021-07-12 at 1 27 33 PM](https://user-images.githubusercontent.com/31771087/125330574-f4274f00-e314-11eb-86ed-96c1c63ec50d.png)